jetty-util/src/main/java/org/eclipse/jetty/util/resource/ResourceCollection.java

```diff
@@ -300,7 +300,8 @@ public Resource addPath(String path) throws IOException
         {
             return new ResourceCollection(resources.toArray(new Resource[0]));
         }
-        return null;
+
+        throw new MalformedURLException();
```

Comment:
Is this tested?

Reply:
No, because I can't see how this can ever happen, as either `resource` or `resources` will be non null.
Thus throwing rather than returning null just make sure it is seen as an error if it error does somehow get there